packages/normalize-color/index.js

```diff
@@ -300,162 +289,189 @@ function parsePercentage(str) {
   return int / 100;
 }
 
+function getColorRegExStrings() {
+  return {
+    rgb: 'rgb' + call(NUMBER, NUMBER, NUMBER),
+    rgba:
+      'rgba(' +
+      commaSeparatedCall(NUMBER, NUMBER, NUMBER, NUMBER) +
+      '|' +
+      callWithSlashSeparator(NUMBER, NUMBER, NUMBER, NUMBER) +
+      ')',
+    hsl: 'hsl' + call(NUMBER, PERCENTAGE, PERCENTAGE),
+    hsla:
+      'hsla(' +
+      commaSeparatedCall(NUMBER, PERCENTAGE, PERCENTAGE, NUMBER) +
+      '|' +
+      callWithSlashSeparator(NUMBER, PERCENTAGE, PERCENTAGE, NUMBER) +
+      ')',
+    hwb: 'hwb' + call(NUMBER, PERCENTAGE, PERCENTAGE),
+    hex3: '#([0-9a-fA-F]{1})([0-9a-fA-F]{1})([0-9a-fA-F]{1})',
+    hex4: '#([0-9a-fA-F]{1})([0-9a-fA-F]{1})([0-9a-fA-F]{1})([0-9a-fA-F]{1})',
+    hex6: '#([0-9a-fA-F]{6})',
+    hex8: '#([0-9a-fA-F]{8})',
+  };
+}
+
+const namedColors = {
```

Comment:
Please don't do this. Storing namedColors like this means the JS VM needs a big object in memory tracking all the color values, while a switch statement, as it was before means the color table can remain fully in bytecode, which is lazily paged-in.

Reply:
This change will not be required if we do not need to validate the named color.